internal/tasks/asset_resize.go

```diff
@@ -73,7 +74,8 @@ func (c *Context) PollForAssetResizes() (j Job, returnedError error) {
 	if err != nil {
 		if err == sql.ErrNoRows {
 			logg.Debug("no assets to resize - slowing down...")
-			tx.Rollback() //avoid the log line generated by core.RollbackUnlessCommitted()
+			//nolint:errcheck //avoid the log line generated by core.RollbackUnlessCommitted()
+			tx.Rollback()
```

Comment:
This comment placement is confusing. The comment is supposed to describe `tx.Rollback()`, but its placement suggests that it describes the nolint directive.